vpr/src/route/connection_router.cpp

```diff
@@ -967,6 +967,7 @@ std::unique_ptr<ConnectionRouterInterface> make_connection_router(
     const DeviceGrid& grid,
     const RouterLookahead& router_lookahead,
     const t_rr_graph_storage& rr_nodes,
+    RRGraphView* rr_graph,
```

Comment:
I don't think the RRGraphView should be modified by the connection router, so this should be a const RRGraphView* . 
That would then let you pass a const pointer using
&g_vpr_ctx.device().rr_graph (instead of mutable_device).